File: src/ruby/fib.rb
class Fib 
  def fib(n)
    if n == 0
      return 0
    if n <= 2  
      return 1
    else
      return fib(n-1) + fib(n-2)
    end
  end
end

f = Fib.new
puts f.fib(45)

File: src/c/fib.c
#include "stdio.h"

long fib(int n) {
  if (n < 2) {
    return 1;
  }
  return fib(n-1) + fib(n-2);
}

int main(int argc, char *args[]) {
  printf("%lu", fib(45));
}
